### PR TITLE
Update README_CN.md

### PR DESCRIPTION
**Pull request by purofle**
Make README_CN.md more friendly for Chinese.

### CODE REVIEW

**Comment by vit9696** [comment]
сс @PMheart

File: README_CN.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ AppleALC
...
-- 支持恢复模式与安装模式.
-- 自动加载声卡Codec.
-- 启用Apple官方不支持的编解码器 (无论是内置还是外置的声卡设备).
-- 任意的kext修改补丁.
-- 自定义设备Layout ID号与支持平台.
-- 工作在 SIP 模式 / El Capitan 系统.
-- 当前版本兼容: 10.4-13.
-
-#### 向以下代码贡献人员致谢
+- 在系统安装阶段即可使用数字或模拟音频。
```
Comment (PMheart) [inline README_CN.md:9]: 。to；maybe?

File: README_CN.md [hunk 2]
```diff
@@ -25,17 +25,17 @@ AppleALC
...
+#### 贡献项目以适配更多机型
+如果你想为更多机器适配不同的 Codec，你需要提供你的配置文件。阅读[wiki](https://github.com/vit9696/AppleALC/wiki) 来获取更多信息。
+如果你想要贡献你的代码，可以看到在很多代码文件头部有 AppleDOC 注释供您参考。
 
-#### 更多的支持与讨论(需要绿色上网环境以提升更好的体验)
-[InsanelyMac上的帖子](http://www.insanelymac.com/forum/topic/311293-applealc-—-dynamic-applehda-patching/) in English  
-[AppleLife上的帖子](https://applelife.ru/threads/applealc-dinamicheskij-patching-applehda.1171672/) in Russian
+#### 更多支持与讨论
+[InsanelyMac上的帖子（英语）](http://www.insanelymac.com/forum/topic/311293-applealc-—-dynamic-applehda-patching/) 
+[AppleLife上的帖子（韩语）](https://applelife.ru/threads/applealc-dinamicheskij-patching-applehda.1171672/) in Russian
```
Comment (PMheart) [inline README_CN.md:38]: 俄语

Comment (purofle) [reply README_CN.md:38]: fixed

File: README_CN.md [hunk 3]
```diff
@@ -34,8 +34,8 @@ AppleALC
 如果你想要贡献你的代码，可以看到在很多代码文件头部有 AppleDOC 注释供您参考。
 
 #### 更多支持与讨论
-[InsanelyMac上的帖子（英语）](http://www.insanelymac.com/forum/topic/311293-applealc-—-dynamic-applehda-patching/) 
-[AppleLife上的帖子（韩语）](https://applelife.ru/threads/applealc-dinamicheskij-patching-applehda.1171672/) in Russian
+[InsanelyMac上的帖子（英语）](http://www.insanelymac.com/forum/topic/311293-applealc-—-dynamic-applehda-patching/)
```
Comment (PMheart) [inline README_CN.md:37]: Trailing space, please trim.

**Comment by Andrey1970AppleLife** [comment]
Please synchronize the translation with the English version.

**Comment by Andrey1970AppleLife** [comment]
The Chinese version differs from English, check and synchronize from the English.

File: README_CN.md [hunk 4]
```diff
@@ -3,39 +3,42 @@ AppleALC
...
-- 启用Apple官方不支持的编解码器 (无论是内置还是外置的声卡设备).
-- 任意的kext修改补丁.
-- 自定义设备Layout ID号与支持平台.
-- 工作在 SIP 模式 / El Capitan 系统.
-- 当前版本兼容: 10.4-13.
-
-#### 向以下代码贡献人员致谢
+- 在系统安装阶段即可使用数字或模拟音频
+- 支持恢复模式与安装模式
+- 自动加载声卡 Codec
```
Comment (PMheart) [inline README_CN.md:14]: 自动检测

File: README_CN.md [hunk 5]
```diff
@@ -3,39 +3,42 @@ AppleALC
...
+- 启用 Apple 官方不支持的编解码器 (无论是内置还是外置的声卡设备)
+- 任意的 Kext 修改补丁
+- 可自定义设备的 Layout ID 号与支持平台
```
Comment (PMheart) [inline README_CN.md:17]: The term `platform` does not have to be translated in my opinion.

File: README_CN.md [hunk 6]
```diff
@@ -3,39 +3,42 @@ AppleALC
...
+- 工作在 SIP 模式 / El Capitan 系统
```
Comment (PMheart) [inline README_CN.md:18]: El Capitan+

**Comment by PMheart** [comment]
Thanks!

**Pull request merged**